modules/ROOT/pages/platform/security/single-sign-on.adoc

```diff
@@ -5,30 +5,112 @@
+
+== Configuring SSO 
+
+Users can configure a combination of one or more of the following login methods:
+
+* Email/password
+* Okta
+* Microsoft Entra ID (previously Azure Active Directory)
+
+An organization's admin can add Aura as a login from a tile in an organization's Apps Dashboard.
+
+
+== SSO setup requirements 
+
+To configure SSO from Aura, go to the *Aura Console > Settings > SSO Configuration.*
+
+The set-up form requires the following info:
+
+* SSO config ID - you decide a name for this
+* IdP type (e.g. Okta / Microsoft Entra ID)
+* Channel
+* Client ID 
+* Client Secret
+* Discovery URI
+* Issuer
+* Authorization Endpoint
+
+== Organization SSO
+
+Organization SSO allows log in to the Aura Console using organization IdP credentials.
+
+To facilitate organization SSO login, you need to create an Auth0 organization and add all users to it.
+Then you need to associate tenants/projects with an organization and add the users to it when they accept invitations to join.
+
+== Mapping an organization
+
+When you map an organization to your IdP, Aura grants membership in that organization to users who authenticate through the IdP .
+
+You can give these users a default role in the selected organization.
+
+Organization mapping lets you configure a single IdP to grant users access to multiple organizations within the Neo4j Aura Console. 
+
+.Organization roles
+[cols="25m,75v"]
+|===
+| Role | Permissions
+
+| Organization Owner
+| • Full access to read/write organization
+
+• Full access to list/read/write tenants
+
+• Given to self serve users during registration instead of a tenant role
+
+• Can create/edit/delete SSO [Enterprise Only]
+
+• Can invite to any role up to owner
+
+| Organization Admin
+| • Full access to read/write org details
+
+• Full access to list/create tenants
+
+• Can create/edit/delete SSO [Enterprise Only]
+
+• Can invite to any role up to admin
+
+| Organization Member (Default when added to tenant)
+| • Full access to read organization
+
+• Full access to list tenants
+
+• Default role for users given access to a single tenant via invite/marketplace
+
+|===
+
+== Home Realm Discovery 
+
+If Home Realm Discovery (HRD) is enabled, and a user’s email domain matches the configured domain, they are redirected to the Auth0 connection IdP. 
+Once authenticated, the user has access to both the company organization and their personal organization. 
+If HRD is disabled, the user logs in with their password, but only has access to their personal organization, once authenticated.
 
 == Console SSO
 
-Console SSO allows you to log in to the Aura Console using company IdP credentials and grants link:{neo4j-docs-base-uri}/cypher-manual/current/administration/access-control/built-in-roles#access-control-built-in-roles-public[Public Access privileges] to all instances in the tenant.
+Console SSO allows log in to the Aura Console using company IdP credentials and grants Public Access privileges to all instances in the tenant.
```

Comment:
This seems quite confusing, isn't this just instance level sso? 

Console/Instance sso does not allow them to log in to the aura console, thats what organization sso does. Instance/console sso can be configured without organization sso, so users _could_ be required to login to console with email/pw but they can use sso to access instnaces.

Reply:
So reading a bit further, there is a section on instance sso using the old process of going through support. Console SSO and Instance SSO are the same, we configure the same stuff. The main difference is that support has more control of the sso configuration as they can do special stuff manually in the raw neo4j.conf file. Our goal is to bridge the gap between what users can configure in the UI and what they can configure via support. But they are the same feature.

I think it might be better to establish the feature gaps between what the customer can do for console/instance sso and what support can do, so they know when to reach out to support for a particular feature. Off the top of my head those features are:
- Role mapping specific to a database
- custom groups claim besides `groups`
- Updating sso of already running instances.

Maybe support has more they can add here